tests/NRedisStack.Tests/NRedisStackConfigurationTests.cs

```diff
@@ -0,0 +1,23 @@
+using Xunit;
+using NRedisStack.Core;
+using static NRedisStack.Auxiliary;
+
+
+namespace NRedisStack.Tests.Core;
+
+public class NRedisStackConfigurationTests : AbstractNRedisStackTest, IDisposable
+{
+    public NRedisStackConfigurationTests(RedisFixture redisFixture) : base(redisFixture) { }
+
+    // TODO: add tests
+
+    [SkipIfRedis(Is.OSSCluster, Comparison.LessThan, "7.1.242")]
+    public void TestDefaultLibName()
```

Comment:
IMHO this should be RedisUriParserTests, to align with the other class. Simlarly, let's add a mass coverage matrix. Can something [like this](https://stackoverflow.com/a/13710788) work?